lib/policy_machine_storage_adapters/active_record/postgresql.rb

```diff
@@ -2,6 +2,127 @@
 
 module PolicyMachineStorageAdapter
   class ActiveRecord
+    class PolicyElementAssociation
+      def self.all_accessible_objects(associations, root_id:, filters: {})
+        pea_ids = associations.pluck(:id)
+        return [] if pea_ids.empty?
```

Comment:
As is, this can return different _types_ of objects – either an active record association of policy elements or arrays. If `.pluck` returns an active record association, then can we return that here to be consistent? Like...
```ruby
return pea_ids if pea_ids.empty?
```

Reply:
how about the ole `PolicyElement.none`